src/app/api/add-pet/route.ts

```diff
@@ -0,0 +1,18 @@
+import { sql } from "@vercel/postgres"
```

Comment:
Dummy route. Api routes using serverless nextjs means we have to create endpoint folder structures. So, for example, adding a pet would be a directory like so: `src/app/api/[endpoint]/route.ts` or in this case `src/app/api/add-pet/route.ts`

All routes must have a route file. We use typescript in our app so our route file is ts.